Only use boost::python namespace after Magick++ headers have been included.

2011-03-15 Martin von Gagern

References:
http://bugs.gentoo.org/358355

Index: PythonMagick-0.9.5/pythonmagick_src/_Drawable.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_Drawable.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_Drawable.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 // Module ======================================================================
 void Export_pyste_src_Drawable()
 {
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableAffine.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableAffine.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableAffine.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableAffine_Wrapper: Magick::DrawableAffine
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableArc.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableArc.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableArc.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableArc_Wrapper: Magick::DrawableArc
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableBezier.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableBezier.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableBezier.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableBezier_Wrapper: Magick::DrawableBezier
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableCircle.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableCircle.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableCircle.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableCircle_Wrapper: Magick::DrawableCircle
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableClipPath.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableClipPath.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableClipPath.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableClipPath_Wrapper: Magick::DrawableClipPath
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableColor.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableColor.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableColor.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableColor_Wrapper: Magick::DrawableColor
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableCompositeImage.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableCompositeImage.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableCompositeImage.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableCompositeImage_Wrapper: Magick::DrawableCompositeImage
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableDashArray.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableDashArray.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableDashArray.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableDashArray_Wrapper: Magick::DrawableDashArray
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableDashOffset.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableDashOffset.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableDashOffset.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableDashOffset_Wrapper: Magick::DrawableDashOffset
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableEllipse.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableEllipse.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableEllipse.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableEllipse_Wrapper: Magick::DrawableEllipse
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableFillColor.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableFillColor.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableFillColor.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableFillColor_Wrapper: Magick::DrawableFillColor
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableFillOpacity.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableFillOpacity.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableFillOpacity.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableFillOpacity_Wrapper: Magick::DrawableFillOpacity
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableFillRule.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableFillRule.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableFillRule.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableFillRule_Wrapper: Magick::DrawableFillRule
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableFont.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableFont.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableFont.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableFont_Wrapper: Magick::DrawableFont
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableGravity.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableGravity.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableGravity.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableGravity_Wrapper: Magick::DrawableGravity
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableLine.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableLine.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableLine.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableLine_Wrapper: Magick::DrawableLine
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableMatte.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableMatte.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableMatte.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableMatte_Wrapper: Magick::DrawableMatte
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableMiterLimit.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableMiterLimit.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableMiterLimit.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableMiterLimit_Wrapper: Magick::DrawableMiterLimit
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePath.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePath.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePath.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePath_Wrapper: Magick::DrawablePath
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePoint.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePoint.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePoint.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePoint_Wrapper: Magick::DrawablePoint
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePointSize.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePointSize.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePointSize.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePointSize_Wrapper: Magick::DrawablePointSize
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePolygon.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePolygon.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePolygon.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePolygon_Wrapper: Magick::DrawablePolygon
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePolyline.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePolyline.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePolyline.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePolyline_Wrapper: Magick::DrawablePolyline
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePopClipPath.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePopClipPath.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePopClipPath.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePopClipPath_Wrapper: Magick::DrawablePopClipPath
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePopGraphicContext.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePopGraphicContext.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePopGraphicContext.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePopGraphicContext_Wrapper: Magick::DrawablePopGraphicContext
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePopPattern.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePopPattern.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePopPattern.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePopPattern_Wrapper: Magick::DrawablePopPattern
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePushClipPath.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePushClipPath.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePushClipPath.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePushClipPath_Wrapper: Magick::DrawablePushClipPath
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePushGraphicContext.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePushGraphicContext.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePushGraphicContext.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePushGraphicContext_Wrapper: Magick::DrawablePushGraphicContext
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawablePushPattern.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawablePushPattern.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawablePushPattern.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawablePushPattern_Wrapper: Magick::DrawablePushPattern
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableRectangle.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableRectangle.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableRectangle.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableRectangle_Wrapper: Magick::DrawableRectangle
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableRotation.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableRotation.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableRotation.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableRotation_Wrapper: Magick::DrawableRotation
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableRoundRectangle.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableRoundRectangle.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableRoundRectangle.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableRoundRectangle_Wrapper: Magick::DrawableRoundRectangle
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableScaling.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableScaling.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableScaling.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableScaling_Wrapper: Magick::DrawableScaling
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableSkewX.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableSkewX.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableSkewX.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableSkewX_Wrapper: Magick::DrawableSkewX
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableSkewY.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableSkewY.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableSkewY.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableSkewY_Wrapper: Magick::DrawableSkewY
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeAntialias.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableStrokeAntialias.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeAntialias.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableStrokeAntialias_Wrapper: Magick::DrawableStrokeAntialias
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeColor.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableStrokeColor.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeColor.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableStrokeColor_Wrapper: Magick::DrawableStrokeColor
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeLineCap.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableStrokeLineCap.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeLineCap.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableStrokeLineCap_Wrapper: Magick::DrawableStrokeLineCap
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeLineJoin.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableStrokeLineJoin.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeLineJoin.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableStrokeLineJoin_Wrapper: Magick::DrawableStrokeLineJoin
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeOpacity.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableStrokeOpacity.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeOpacity.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableStrokeOpacity_Wrapper: Magick::DrawableStrokeOpacity
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeWidth.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableStrokeWidth.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableStrokeWidth.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableStrokeWidth_Wrapper: Magick::DrawableStrokeWidth
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableText.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableText.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableText.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableText_Wrapper: Magick::DrawableText
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableTextAntialias.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableTextAntialias.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableTextAntialias.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableTextAntialias_Wrapper: Magick::DrawableTextAntialias
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableTextDecoration.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableTextDecoration.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableTextDecoration.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableTextDecoration_Wrapper: Magick::DrawableTextDecoration
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableTextUnderColor.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableTextUnderColor.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableTextUnderColor.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableTextUnderColor_Wrapper: Magick::DrawableTextUnderColor
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableTranslation.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableTranslation.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableTranslation.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableTranslation_Wrapper: Magick::DrawableTranslation
Index: PythonMagick-0.9.5/pythonmagick_src/_DrawableViewbox.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_DrawableViewbox.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_DrawableViewbox.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_DrawableViewbox_Wrapper: Magick::DrawableViewbox
Index: PythonMagick-0.9.5/pythonmagick_src/_PathArcAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathArcAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathArcAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathArcAbs_Wrapper: Magick::PathArcAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathArcArgs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathArcArgs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathArcArgs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 // Module ======================================================================
 void Export_pyste_src_PathArcArgs()
 {
Index: PythonMagick-0.9.5/pythonmagick_src/_PathArcRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathArcRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathArcRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathArcRel_Wrapper: Magick::PathArcRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathClosePath.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathClosePath.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathClosePath.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathClosePath_Wrapper: Magick::PathClosePath
Index: PythonMagick-0.9.5/pythonmagick_src/_PathCurvetoAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathCurvetoAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathCurvetoAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathCurvetoAbs_Wrapper: Magick::PathCurvetoAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathCurvetoArgs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathCurvetoArgs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathCurvetoArgs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 // Module ======================================================================
 void Export_pyste_src_PathCurvetoArgs()
 {
Index: PythonMagick-0.9.5/pythonmagick_src/_PathCurvetoRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathCurvetoRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathCurvetoRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathCurvetoRel_Wrapper: Magick::PathCurvetoRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathLinetoAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathLinetoAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathLinetoAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathLinetoAbs_Wrapper: Magick::PathLinetoAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathLinetoHorizontalAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathLinetoHorizontalAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathLinetoHorizontalAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathLinetoHorizontalAbs_Wrapper: Magick::PathLinetoHorizontalAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathLinetoHorizontalRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathLinetoHorizontalRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathLinetoHorizontalRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathLinetoHorizontalRel_Wrapper: Magick::PathLinetoHorizontalRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathLinetoRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathLinetoRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathLinetoRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathLinetoRel_Wrapper: Magick::PathLinetoRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathLinetoVerticalAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathLinetoVerticalAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathLinetoVerticalAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathLinetoVerticalAbs_Wrapper: Magick::PathLinetoVerticalAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathLinetoVerticalRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathLinetoVerticalRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathLinetoVerticalRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathLinetoVerticalRel_Wrapper: Magick::PathLinetoVerticalRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathMovetoAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathMovetoAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathMovetoAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathMovetoAbs_Wrapper: Magick::PathMovetoAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathMovetoRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathMovetoRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathMovetoRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathMovetoRel_Wrapper: Magick::PathMovetoRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathQuadraticCurvetoAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathQuadraticCurvetoAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathQuadraticCurvetoAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathQuadraticCurvetoAbs_Wrapper: Magick::PathQuadraticCurvetoAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathQuadraticCurvetoArgs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathQuadraticCurvetoArgs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathQuadraticCurvetoArgs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 // Module ======================================================================
 void Export_pyste_src_PathQuadraticCurvetoArgs()
 {
Index: PythonMagick-0.9.5/pythonmagick_src/_PathQuadraticCurvetoRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathQuadraticCurvetoRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathQuadraticCurvetoRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathQuadraticCurvetoRel_Wrapper: Magick::PathQuadraticCurvetoRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathSmoothCurvetoAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathSmoothCurvetoAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathSmoothCurvetoAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathSmoothCurvetoAbs_Wrapper: Magick::PathSmoothCurvetoAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathSmoothCurvetoRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathSmoothCurvetoRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathSmoothCurvetoRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathSmoothCurvetoRel_Wrapper: Magick::PathSmoothCurvetoRel
Index: PythonMagick-0.9.5/pythonmagick_src/_PathSmoothQuadraticCurvetoAbs.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathSmoothQuadraticCurvetoAbs.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathSmoothQuadraticCurvetoAbs.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathSmoothQuadraticCurvetoAbs_Wrapper: Magick::PathSmoothQuadraticCurvetoAbs
Index: PythonMagick-0.9.5/pythonmagick_src/_PathSmoothQuadraticCurvetoRel.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_PathSmoothQuadraticCurvetoRel.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_PathSmoothQuadraticCurvetoRel.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 namespace  {
 
 struct Magick_PathSmoothQuadraticCurvetoRel_Wrapper: Magick::PathSmoothQuadraticCurvetoRel
Index: PythonMagick-0.9.5/pythonmagick_src/_VPath.cpp
===================================================================
--- PythonMagick-0.9.5.orig/pythonmagick_src/_VPath.cpp
+++ PythonMagick-0.9.5/pythonmagick_src/_VPath.cpp
@@ -6,12 +6,12 @@
 // Includes ====================================================================
 #include <Magick++/Drawable.h>
 
-// Using =======================================================================
-using namespace boost::python;
-
 // Declarations ================================================================
 #include <Magick++.h>
 
+// Using =======================================================================
+using namespace boost::python;
+
 // Module ======================================================================
 void Export_pyste_src_VPath()
 {
